### PR TITLE
Added x-mirror bone feature

### PR DESCRIPTION
**Pull request by sts1skj**
Added a GUI element that appears when the bones tab is selected that allows
the user to select an x-mirror bone.  Changes by any of the three brushes
(weight, unweight, smooth weight) are x-mirrored onto the x-mirror bone.
There is also an "Auto" selection for the x-mirror bone.

### CODE REVIEW

File: src/program/OutfitStudio.cpp [hunk 1]
```diff
@@ -869,7 +869,11 @@ OutfitStudioFrame::OutfitStudioFrame(const wxPoint& pos, const wxSize& size) {
 		lightDirectional2Slider->SetValue(directional2);
 	}
 
+	wxPanel* m_panel6 = (wxPanel*)FindWindowByName("m_panel6");
+	m_panel6->SetBackgroundColour(wxColour(112, 112, 112));
```
Comment (ousnius) [inline src/program/OutfitStudio.cpp]: Why is this required? Is there a way to do it more cleanly (in the XRC maybe)?

Comment (sts1skj) [reply src/program/OutfitStudio.cpp]: I tried setting m_panel6's background color in the xrc, and wxWidgets for gtk ignored it.  Without the background color being set, all of the text labels in the panel are white on an off-white background for me.  (That's "De-/Select Sliders", "Fixed Weight Brush", "X-Mirror Bone", and "Preview Scaling".)

Comment (sts1skj) [reply src/program/OutfitStudio.cpp]: The parent panel of m_panel6 is bottomSplitPanel.  I tried setting bottomSplitPanel's background color instead to see if the color propagates to the child panel, and it doesn't.  The panel that needs its color set is m_panel6.

**Comment by ousnius** [comment]
I gave it a quick spin and it seems to work very well so far, thank you!
